### PR TITLE
Add pack installation instructions

### PR DESCRIPTION
**Pull request by lucperkins**
I sadly don't really have the chops to add Windows installation docs. I'll leave that as an exercise for others.

I also added a reusable shortcode that can infer the latest version using the GitHub Releases API.

### CODE REVIEW

**Comment by lucperkins** [comment]
Any chance I could get a review on this? It’d be really nice to get installation instructions.

File: src/content/docs/using-pack/install-pack.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+tar xvf pack-{{< latest >}}-linux.tar.gz
+pack-{{< latest >}}-linux.tar.gz
+./pack --help
+```
+
+From there, you can copy the executable to a directory like `/usr/local/bin` or add the current directory to your `PATH`.
+
+## Windows
+
+You can install the Windows executable for `pack` by downloading the Windows [ZIP file](https://github.com/buildpack/pack/releases/download/v{{< latest >}}/pack-{{< latest >}}-windows.zip).
```
[the remark this answers is not in the document]
Comment (lucperkins) [reply src/content/docs/using-pack/install-pack.md]: I'm unfortunately not knowledgeable enough about Windows to provide this

**Comment by lucperkins** [comment]
@sclevine Question: in order to publish a newer version of the site, do I need to re-generate the `docs` folder in this PR?

**Comment by sclevine** [comment]
Re-generated in d255bba2c9cfdad5a3cb354031227c7f6a0597ed. Thanks for the contribution, and apologies again for the delay.

**Pull request merged**